config/permissions.tmpl.yml

```diff
@@ -36,6 +36,7 @@
 - ems-type:openstack_infra
 - ems-type:openstack_network
 - ems-type:physical_infra_manager
+- ems-type:lenovo_ph_infra
```

Comment:
I see, problem was it keep braking the tests, as these text strings dotted around everywhere where all getting out of sync/in the wrong place. Thanks for heads up. Seems to be working now.